### ISSUE TITLE
USER ISSUE: Connection to server timed out, other players can join server

### ISSUE THREAD

**Issue by sabr1993** [StrangeLoopGames/EcoIssues#19394]
\
**Version:** 0.9.1.5 beta release-119

**Steps to Reproduce:**
1. Hosted private server
2. joined server one time
3. closed game and wanted to play next day
4. tried to connect 
5. can't connect to server, meanwhile my friend can

**Expected behavior:**
I should have joined the server, the server is online, my friend can play, but I can't

**Actual behavior:**
Die folgende Fehlermeldung trat auf:
```
<size=150%>Verbindung fehlgeschlagen</size>

Fehler beim Verbinden - Verbindungszeit überschritten.
```
I don't see the ping to the server, I can only join the official ECO servers

**Do you have mods installed? Does the issue happen when no mods are installed?:**
no mods

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
@sabr1993 Thank you for the report and sorry for thw waiting.
1. Can you provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
2. Do you have Firewall or Antivirus enabled?
3. Which kind of internet connection are you using?
4. Please provide your PC specification (include Operating System).

**Comment by sabr1993** [comment]
@SlayksWood thank you for responding. I have further information for you. It seems that it has something to do with the language option the client is using. I played with german localization and I got always the described connection problems. The server runs with english localization. Now I switched the language to german and the problem has disappeared.

And now to your questions:
1. At the moment I cannot provide log files with the specific bug, I tried to reproduce it but today it seems to work. If this changes, I will attach logfiles at this issue.
2. I am using the standard windows 10 firewall and windows defender. I have no extra software installed.
3. I am connected to internet through ethernet cable.
4. PC specs are listed below.
5. IP adress from server is 27.120.171.146:3000

PC specs:
msi Z97 Gaming 3 motherboard
Intel Core i7-4790K CPU 4.0GHz
16 GB RAM
NVIDIA GeForce GTX 1060 6GB
OS: Windows 10 64 bit
